example_dags/__init__.py

```diff
@@ -1,15 +0,0 @@
-"""
-Copyright Astronomer, Inc.
```

Comment:
This licenses are sporadic, either we have them in all files or only in LICENSE. Hence I have removed